src/eredis_client.erl

```diff
@@ -230,13 +231,14 @@ handle_info(initiate_connection, #state{socket = undefined} = State) ->
         {ok, NewState} ->
             {noreply, NewState};
         {error, Reason} ->
-            {noreply, schedule_reconnect(Reason, State)}
+            ?LOG_WARNING("eredis: Initial connect failed: ~p", [Reason]),
```

Comment:
```suggestion
            ?LOG_WARNING("Initial connect failed: ~p", [Reason]),
```

Removing `eredis`, can be given via metadata [mfa](https://www.erlang.org/doc/man/logger.html#type-metadata)